### PR TITLE
week1-html fundamentals

### PR DESCRIPTION
**Pull request by Navya-Dandu**
week2-assignment

### CODE REVIEW

File: Week-2/article/article.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link href="styles.css" rel="stylesheet">
+    <title>5 Things to Know About Rattlesnakes and Their Babies</title>
+</head>
+<body>
+    <img class="first_image" src="https://static-resources.zybooks.com/static/UofA_header.svg">
```
Comment (manishsundriyal) [inline Week-2/article/article.html:8]: Always add alt text property for an img tag
https://www.w3schools.com/tags/att_img_alt.asp

File: Week-2/article/styles.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
+img {
+    width: 300px;
+}
+
+.first_image {
```
Comment (manishsundriyal) [inline Week-2/article/styles.css:5]: any class/id/variable name should be meaningful. 

.logo would be a better class name in this case

File: Week-2/fav_musician.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+</head>
+<body>
+    <h1>Depeche Mode Fan Webpage</h1>
+    <main>
+<p>Depeche Mode is an English electronic band that formed in 1980. The band members are currently Dave Gahan, 
+    Martin Gore, and Andy Fletcher.</p>
+<p>Visit the <a href="https://google.com" alt="link" >official website</a> or a 
+    <a href="https://wiki.com">Wikipedia article</a> about the band.</p>
+</main>
+Fan web page by Anonymous</p>
```
Comment (manishsundriyal) [inline Week-2/fav_musician.html:16]: Cannot find the start tag for this `</p>` tag

File: Week-2/table1.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Table1</title>
+</head>
+<body>
+    <table border="1px solid black">
```
Comment (manishsundriyal) [inline Week-2/table1.html:10]: the table tag only accepts integer for border attribute: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table#attributes
```suggestion
    <table border="1">
```

**Review by manishsundriyal — CHANGES_REQUESTED** [review]

File: Week-2/article/styles.css (hunk 2 above)
Comment (Navya-Dandu) [reply Week-2/article/styles.css:5]: okay

**Pull request merged**